ir-sanguinem-et-lacrimis/common/technologies/artillery.txt

```diff
@@ -1855,14 +1857,13 @@ technologies = {
 		categories = {
 			artillery
 			cat_anti_tank
+			rocketry
 		}
 	}
 
 	atgm_1 = {
+	#	enable_equipments = { mot_atgm_equipment_1 }
 
-	#	enable_equipments = {
-	#		mot_atgm_equipment_1
-	#	}
```

Comment:
has actually nothing to do with the bureau and tactics folder; still cleaning up some old gunk is always good